### PR TITLE
New package: firehol-3.1.6

### PR DESCRIPTION
**Pull request by juantascon**

### CODE REVIEW

**Comment by ndowens** [comment]
Looks like you are missing xmllint

**Comment by juantascon** [comment]
Thanks for the input, I was trying different combinations because I had an issue while building for aarch64 and since I'm new to void I didn't know how to cross compile locally. I think I've figured it out

**Comment by juantascon** [comment]
@ndowens could you please review

File: srcpkgs/firehol/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'firehol'
+pkgname=firehol
+version=3.1.6
+revision=1
+build_style=gnu-configure
+configure_args="--disable-doc --disable-man"
+hostmakedepends="automake iproute2 kmod procps-ng iprange iputils traceroute curl ipset"
+#makedepends="pandoc"
```
Comment (ndowens) [inline srcpkgs/firehol/template]: Remove unused comment

Comment (juantascon) [reply srcpkgs/firehol/template]: done!

**Comment by juantascon** [comment]
@ndowens could you please review again. In the future, is there anything I can do to make sure my pullrq doesn't just die among all of those other open pullrqs?

**Comment by ndowens** [comment]
> @ndowens could you please review again. In the future, is there anything I can do to make sure my pullrq doesn't just die among all of those other open pullrqs?

I don’t know any tip besides wait, I can’t commit myself either

**Pull request merged**